docs/faq.md

```diff
@@ -188,10 +205,10 @@ If you choose to enable it, Semgrep Assistant requires code access. See the [Pri
 
 [Semgrep](https://github.com/semgrep/semgrep) sends data to Semgrep AppSec Platform in accordance with the [metrics policy](/metrics).
 
-These types of data are **scan data** and **findings data**.
+These types of data include **scan data** and **findings data**.
 
 - Scan data includes project name, CI environment, and scan meta-data.
-- Findings data are used to provide human-readable content for notifications and integrations, as well as tracking results as new, fixed, or duplicate.
+- Findings data is used to provide human-readable content for notifications and integrations, as well as tracking results as new, fixed, or duplicate.
```

Comment:
```suggestion
- Findings data are used to provide human-readable content for notifications and integrations, as well as tracking results as new, fixed, or duplicate.
```

Reply:
Note to self: turn off the Vale rule for this.